test/Interop/Cxx/foreign-reference/reference-counted-objc-property.swift

```diff
@@ -0,0 +1,27 @@
+// RUN: %target-swift-emit-irgen %s -I %S/Inputs -cxx-interoperability-mode=default -Xcc -fno-exceptions -Xcc -fno-objc-exceptions | %FileCheck %s
+
+// REQUIRES: OS=macosx
```

Comment:
Could you please change this to `// REQUIRES: objc_interop` to make this test also run on iOS and other Apple platforms?